pims/tests/test_bioformats.py

```diff
@@ -541,6 +541,37 @@ def test_metadata_raw(self):
         assert_equal(metadata['CH2ChannelDyeName'], '5-FAM/pH 9.0')
         assert_equal(metadata['dCalibration'], '0.16780898323268245')
 
+    def test_metadata_tags(self):
+        self.v = self.klass(self.filename, meta=True)
+        fields = self.v.metadata.fields
+        expected_fields = {
```

Comment:
I think this test is easily broken with different Bioformats versions.
Maybe you can only test a few? Something like `assert 'PixelsPhysicalSizeX' in fields`